src/parser/getCSSRules.ts

```diff
@@ -23,39 +24,40 @@ export function getCSSRules(doc: Document) {
     }
     // recovers all the rule in this form `body { style code... }`
     // rules = styleContents.match(/[^{]*\{[\s\S]*?\}/g);
-    rules = styleContents.split('}');
-    // remove empty rules.
-    rules = rules.filter(function (rule) {
-      return rule.trim();
-    });
-    // at this point we have hopefully an array of rules `body { style code... `
-    // eslint-disable-next-line no-loop-func
-    rules.forEach(function (rule) {
-      const match = rule.split('{'),
-        ruleObj: Record<string, string> = {},
-        declaration = match[1].trim(),
-        propertyValuePairs = declaration.split(';').filter(function (pair) {
-          return pair.trim();
-        });
+    styleContents
+      .split('}')
+      // remove empty rules.
+      .filter(function (rule) {
+        return rule.trim();
+      })
+      // at this point we have hopefully an array of rules `body { style code... `
+      // eslint-disable-next-line no-loop-func
+      .forEach(function (rule) {
+        const match = rule.split('{'),
+          ruleObj: Record<string, string> = {},
+          declaration = match[1].trim(),
+          propertyValuePairs = declaration.split(';').filter(function (pair) {
+            return pair.trim();
+          });
 
-      for (i = 0, len = propertyValuePairs.length; i < len; i++) {
-        const pair = propertyValuePairs[i].split(':'),
-          property = pair[0].trim(),
-          value = pair[1].trim();
-        ruleObj[property] = value;
-      }
-      rule = match[0].trim();
-      rule.split(',').forEach((_rule) => {
-        _rule = _rule.replace(/^svg/i, '').trim();
-        if (_rule === '') {
-          return;
+        for (i = 0, len = propertyValuePairs.length; i < len; i++) {
+          const pair = propertyValuePairs[i].split(':'),
+            property = pair[0].trim(),
+            value = pair[1].trim();
+          ruleObj[property] = value;
         }
-        allRules[_rule] = {
-          ...(allRules[_rule] || {}),
-          ...ruleObj,
-        };
+        rule = match[0].trim();
+        rule.split(',').forEach((_rule) => {
+          _rule = _rule.replace(/^svg/i, '').trim();
+          if (_rule === '') {
+            return;
+          }
+          allRules[_rule] = {
+            ...(allRules[_rule] || {}),
+            ...ruleObj,
+          };
+        });
       });
-    });
```

Comment:
looks bad but i really just removed the var  `rules` and chained 3 array methods of which we didn't need the return value